fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dto/DateParmVal.java

```diff
@@ -50,13 +52,22 @@ public void setValueDateEnd(Timestamp valueDateEnd) {
     /**
      * We know our type, so we can call the correct method on the visitor
      */
+    @Override
     public void accept(ExtractedParameterValueVisitor visitor) throws FHIRPersistenceException {
         visitor.visit(this);
     }
 
+    @Override
+    public String getHash(ParameterHashUtil parameterHashUtil) {
+        StringBuilder sb = new StringBuilder();
```

Comment:
I don't think we should be using strings to build the hash. Rendering these values as strings is going to consume more CPU than necessary. It should be possible to accumulate the hash directly from the raw Java types.

Reply:
I think I need more details on exactly how you'd like me to generate the hash.  Can walk through it, so I can understand better.

Reply:
No longer using strings to build the hash.